Sources/PointFree/Emails/NewEpisodeEmail.swift

```diff
@@ -10,59 +10,94 @@ import Prelude
+          ? subscriberAnnouncement
+          : nonSubscriberAnnouncement,
+        user.subscriptionId != nil
+      )
     )
 }
 
-let newEpisodeEmailContent = View<(Episode, isSubscriber: Bool)> { ep, isSubscriber in
+let newEpisodeEmailContent = View<(Episode, String?, isSubscriber: Bool)> { ep, announcement, isSubscriber in
   emailTable([style(contentTableStyles)], [
     tr([
       td([valign(.top)], [
-        div([`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])], [
-          a([href(url(to: .episode(.left(ep.slug))))], [
-            h3([`class`([Class.pf.type.responsiveTitle3])], [text("#\(ep.sequence): \(ep.title)")]),
-            ]),
-          p([.text(encode(ep.blurb))]),
-          p([`class`([Class.padding([.mobile: [.topBottom: 2]])])], [
-            a([href(url(to: .episode(.left(ep.slug))))], [
-              img(src: ep.image, alt: "", [style(maxWidth(.pct(100)))])
-              ])
-            ])
-          ]
-          <> nonSubscriberCtaView.view((ep, isSubscriber))
-          <> subscriberCtaView.view((ep, isSubscriber))
-          <> hostSignOffView.view(unit))
+        div([`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])],
+
+            announcementView.view(announcement) <> [
+
+              a([href(url(to: .episode(.left(ep.slug))))], [
+                h3([`class`([Class.pf.type.responsiveTitle3])], [text("#\(ep.sequence): \(ep.title)")]),
+                ]),
+              p([.text(encode(ep.blurb))]),
+              p([`class`([Class.padding([.mobile: [.topBottom: 2]])])], [
+                a([href(url(to: .episode(.left(ep.slug))))], [
+                  img(src: ep.image, alt: "", [style(maxWidth(.pct(100)))])
+                  ])
+                ])
+              ]
+              <> nonSubscriberCtaView.view((ep, isSubscriber))
+              <> subscriberCtaView.view((ep, isSubscriber))
+              <> hostSignOffView.view(unit))
         ])
       ])
     ])
 }
 
+private let announcementView = View<String?> { announcement -> [Node] in
+  guard let announcement = announcement, !announcement.isEmpty else { return [] }
+
+  return [
+    blockquote(
+      [
+        `class`(
+          [
+            Class.padding([.mobile: [.all: 2]]),
+            Class.margin([.mobile: [.leftRight: 0, .topBottom: 3]]),
+            Class.pf.colors.bg.blue900,
+            Class.type.italic
+          ]
+        )
+      ],
+      [
+        h5([`class`([Class.pf.type.title5])], ["Announcements"]),
+        markdownBlock(announcement)
+      ]
+    )
+  ]
+}
+
 private let nonSubscriberCtaView = View<(Episode, isSubscriber: Bool)> { ep, isSubscriber -> [Node] in
   guard !isSubscriber else { return [] }
 
+  let blurb = ep.subscriberOnly
+    ? "This episode is for subscribers only. To access it, and all past and future episodes, become a subscriber today!"
+    : "This episode is free for everyone, made possible by our subscribers. Consider becoming a subscriber today!"
```

Comment:
here's the copy I came up with for when our new episodes are free from the beginning.